application/backend/src/main/resources/apps/typerefinery/components/widgets/columnticker/head-libs.html

```diff
@@ -1,2 +1,2 @@
-<script src="/apps/typerefinery/components/widgets/smallticker/script.js"></script>
+<script src="/apps/typerefinery/components/widgets/smallticker/clientlibs/functions.js"></script>
 <link rel="stylesheet" href="/apps/typerefinery/components/widgets/columnticker/columnstyle.css">
```

Comment:
I could not see behavior.js imported.
Where is that?
 
Without that how will the component render?

Reply:
Done